pom.xml

```diff
@@ -120,6 +120,14 @@
 				<groupId>org.jacoco</groupId>
 				<artifactId>jacoco-maven-plugin</artifactId>
 				<version>${jacoco-maven-plugin.version}</version>
+				<configuration>
```

Comment:
@amerkul it's not a good idea to ignore some packages

coverage should be absolute, not exclusively for only some packages

Reply:
ok